### PR TITLE
CA-115552: Add missing data sources in rrdd plugins

### PR DESCRIPTION
**Pull request by koushikcgit**
Add few missing data sources

Signed-off-by: Koushik Chakravarty <koushik.chakravarty@citrix.com>

### CODE REVIEW

**Comment by koushikcgit** [comment]
Adding @johnelse as reviewer

File: rrdp_iostat.ml [hunk 1]
```diff
@@ -422,6 +450,10 @@ module Stats_value = struct
 		let (++) = Int64.add in
 		List.fold_left (fun acc v ->
 			{
+				rd_bytes = acc.rd_bytes ++ v.rd_bytes;
+				rd_avg_usecs = acc.rd_avg_usecs ++ v.rd_avg_usecs;
```
Comment (johnelse) [inline rrdp_iostat.ml]: Should these values be added, or do we just want the maximum?

[Here](https://github.com/xapi-project/xcp-rrdd/commit/f10769a46acd4b9ee17b16bc47461cbee184215b#diff-aae74f219b1c9bcd5b3de6bc44497519L472) it looks like we used to use the maximum.

File: rrdp_iostat.ml [hunk 2]
```diff
@@ -422,6 +450,10 @@ module Stats_value = struct
...
+				wr_bytes = acc.wr_bytes ++ v.wr_bytes;
+				wr_avg_usecs = acc.wr_avg_usecs ++ v.wr_avg_usecs;
```
Comment (johnelse) [inline rrdp_iostat.ml]: Same question as for rd_avg_usecs.

**Comment by johnelse** [comment]
Can we remove the `io_throughput_read` and `io_throughput_write` datasources now? They seem to be the same values but in MiB.

File: rrdp_iostat.ml (hunk 1 above)
Comment (koushikcgit) [inline rrdp_iostat.ml]: This sum is for the sumup function in stats_value module - I think this is probably to get the total of all stats.

**Comment by koushikcgit** [comment]
Removed `io_throughput_read/write`

**Comment by johnelse** [comment]
Do we still need the `io_throughput_read_mb` and `io_throughput_write_mb` fields? I was expecting those to be removed as well.

**Comment by johnelse** [comment]
Superceded by https://github.com/xenserver/rrdd-plugins/pull/27

**Pull request closed without merging**